workshop/exercise-4/README.md

```diff
@@ -13,80 +13,81 @@ You can read more about how [Istio mixer enables telemetry reporting](https://is
 ### Configure Istio to receive telemetry data
 
 1. Verify that the Grafana, Prometheus, ServiceGraph and Jaeger add-ons were installed successfully. All add-ons are installed into the `istio-system` namespace.
-   ```console
-   kubectl get pods -n istio-system
 
-   kubectl get services -n istio-system
-   ```
+```shell
+kubectl get pods -n istio-system
+kubectl get services -n istio-system
+```
 
 2. Configure Istio to automatically gather telemetry data for services that run in the service mesh.
    1. Go back to your v2 directory.
-      ````
-      cd guestbook/v2
-      ````
+
+```shell
+cd guestbook/v2
+```
 
    2. Create a rule to collect telemetry data.
-      ```sh
-      kubectl create -f guestbook-telemetry.yaml
-      ```
-   3. Obtain the guestbook endpoint to access the guestbook.  
-      i. For paid cluster, you can acceess the guestbook via the external IP for your service as guestbook is deployed as a load blanacer service.  Get the EXTERNAL-IP of the guestbook service via output below:
-
-      ```sh
-      kubectl get service guestbook -n default
-      ```
-      
-      ii. For lite cluster, first, get the worker's public IP:
-      ```sh
-      bx cs workers <cluster_name>
-      ```
-      
-          Examples: 
-          ```
-          bx cs workers cluster1
-          ID             Public IP      Private IP      Machine Type        State    Status   Zone    Version   
-          kube-xxx       169.60.87.20   10.188.80.69    u2c.2x4.encrypted   normal   Ready    wdc06   1.9.7_1510*   
-          ```
-   
-      Second, get the node port:
-      ```sh
-      kubectl get svc guestbook -n default
-      ```
-      
-          Examples:
-          ```
-          $ kubectl get svc guestbook -n default
-          NAME        TYPE           CLUSTER-IP     EXTERNAL-IP    PORT(S)        AGE
-          guestbook   LoadBalancer   172.21.134.6   pending        80:31702/TCP   4d
-          ```
-      
-      The node port in above sample output is `169.60.87.20:31702`
+```shell
+kubectl create -f guestbook-telemetry.yaml
+```
+
+   3. Obtain the guestbook endpoint to access the guestbook.
+   i. For paid cluster, you can acceess the guestbook via the external IP for your service as guestbook is deployed as a load blanacer service.  Get the EXTERNAL-IP of the guestbook service via output below:
```

Comment:
I think you may need a blank line before this - the "i." line doesn't appear as a new numeric list when I view the html

Reply:
actually - I wonder if we should use "-" instead, then the md to html converter might indent it under the "3." like it should be. Right now its actually to the left of the "3."

Reply:
actually, maybe it won't given all of the code snippets in there.  We should probably remove the indent on the "1." lines since indenting provides no value in this case - that might then allow us to use "-"

Reply:
Just to keep rambling... :-)  I'm not a fan of numbered lists if we never refer to the numbers directly. So if a bulleted lists works better (due to the code snippets) then maybe we should do that